content/kubermatic/master/startio/cheat_sheets/work_with_secrets/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Work with Secrets using SOPS"
+weight = 50
++++
+
+Install the [sops](https://github.com/mozilla/sops#download) tool locally.
```

Comment:
It would be useful to specify which SOPS versions we use in the pipeline, so folks can get the same version locally.